### DISCUSSION TITLE
logging format rfc5424

### DISCUSSION THREAD

**Discussion by durd** [aristanetworks/avd#3366] [Ideas]
Hi,

I did this backwards and created an Issue feature request (#3365) first and then a discussion. Sorry about that.

Would anyone be interested in support for RFC5424 syslog in AVD?
I've forked and am working on it anyway, seems straight forward but I have yet to look at tests or documentation.
Any suggestions for creating a PR? Except for reading `contributing.md` that is (and doing stuff in the right order).

Thankful for any opinions or feedback!

**Comment by carlbuchmann** [comment]
@durd Thanks for your interest in contributing to the AVD project.

We need an updated contribution guide as many things have changed in the last 5 months since AVD version 4.0
Starting with just an issue is perfect, no need for discussion but here we are :-)

We are always happy to assist people with their first contributions. You can email the maintainers directly, and we will set up a zoom call.
https://avd.arista.com/4.4/docs/contribution/overview.html#governance

Thanks

Carl

**Reply by durd** [reply]
Thank you so much @carlbuchmann! 
I wish I saw your answer before I created a PR, I could have cleared up some trouble I had heh.

I'd say I'm an "junior" amateur or "senior" novice with Git, just this year I started using branches! That said I had some trouble with `git rebase`. I've done this with GitLab and PR:s there before. But I couldn't figure it out with GitHub or pure CLI. I noticed some PR:s hadn't rebased into a single commit so I went ahead.

I also had trouble with tests. But I'll leave that to the PR:s comment section.

**Comment by ClausHolbechArista** [comment]
Closing this, since we will follow up in the PR.
